### PR TITLE
Feature/wifi

### LINKED ISSUE #97: Hotspot Settings > Update Wi-Fi
Pending designs.

Functionality will be the same as old app.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
closes #97

### CODE REVIEW

File: src/features/hotspots/settings/WifiSetup.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import Box from '../../../components/Box'
+import Button from '../../../components/Button'
+import CircleLoader from '../../../components/CircleLoader'
+import Text from '../../../components/Text'
+import TextInput from '../../../components/TextInput'
+import { useConnectedHotspotContext } from '../../../providers/ConnectedHotspotProvider'
+import animateTransition from '../../../utils/animateTransition'
+import { hp } from '../../../utils/layout'
+import useAlert from '../../../utils/useAlert'
+import useKeyboard from '../../../utils/useKeyboardHeight'
```
Comment (allenan) [inline src/features/hotspots/settings/WifiSetup.tsx:12]: should this import be `useKeyboardHeight` instead of just `useKeyboard`?

Comment (matthewcarlreetz) [reply src/features/hotspots/settings/WifiSetup.tsx:12]: yeah. I think so.

**Comment by allenan** [comment]
closes #97

**Review by allenan — APPROVED** [review]
tested it out and works great!

**Pull request merged**